src/System.ComponentModel.TypeConverter/src/System/Drawing/ColorConverter.cs

```diff
@@ -0,0 +1,353 @@
+                            }
+
+                            // Note: ConvertToString will raise exception if value cannot be converted.
+                            args[nArg++] = intConverter.ConvertToString(context, culture, (object)c.R);
+                            args[nArg++] = intConverter.ConvertToString(context, culture, (object)c.G);
+                            args[nArg++] = intConverter.ConvertToString(context, culture, (object)c.B);
+
+                            // Now slam all of these together with the fantastic Join 
+                            // method.
+                            //
+                            return string.Join(sep, args);
+                        }
+                    }
+                }
+                //if (destinationType == typeof(InstanceDescriptor))
+                //{
+                //    MemberInfo member = null;
+                //    object[] args = null;
+
+                //    Color c = (Color)value;
+
+                //    if (c.IsEmpty)
+                //    {
+                //        member = typeof(Color).GetField("Empty");
+                //    }
+                //    else if (c.IsSystemColor)
+                //    {
+                //        member = typeof(SystemColors).GetProperty(c.Name);
+                //    }
+                //    else if (c.IsKnownColor)
+                //    {
+                //        member = typeof(Color).GetProperty(c.Name);
+                //    }
+                //    else if (c.A != 255)
+                //    {
+                //        member = typeof(Color).GetMethod("FromArgb", new Type[] { typeof(int), typeof(int), typeof(int), typeof(int) });
+                //        args = new object[] { c.A, c.R, c.G, c.B };
+                //    }
+                //    else if (c.IsNamedColor)
+                //    {
+                //        member = typeof(Color).GetMethod("FromName", new Type[] { typeof(string) });
+                //        args = new object[] { c.Name };
+                //    }
+                //    else
+                //    {
+                //        member = typeof(Color).GetMethod("FromArgb", new Type[] { typeof(int), typeof(int), typeof(int) });
+                //        args = new object[] { c.R, c.G, c.B };
+                //    }
+
+                //    Debug.Assert(member != null, "Could not convert color to member.  Did someone change method name / signature and not update Colorconverter?");
+                //    if (member != null)
+                //    {
+                //        return new InstanceDescriptor(member, args);
+                //    }
+                //    else
+                //    {
+                //        return null;
+                //    }
+                //}
+            }
+
+            return base.ConvertTo(context, culture, value, destinationType);
+        }
+
+        private static void FillConstants(Hashtable hash, Type enumType)
```

Comment:
Other than the Hashtable/Dictionary difference, this looks the same as the helper used earlier.  Can they be shared?